### ISSUE TITLE
Allow disabling of default remark plugins

### ISSUE THREAD

**Issue by smoores-dev** [typedoc2md/typedoc-plugin-markdown#735]
### What package is the feature request related to?

typdedoc-plugin-remark

### Background

`typedoc-plugin-remark` always adds three remark plugins &mdash; `remark-mdx`, `remark-gfm`, and `remark-frontmatter`. As all of these plugins change the default parsing behavior of remark in one way or another, it would be nice to be able to turn them off when that behavior isn't desired. A simple example: HTML-style comments (`<!-- ... -->`) aren't supported by the MDX parser, but MDX-style comments (`{/* ... */}`) aren't supported by other pure-Markdown tooling. This means that plain markdown files essentially can't have comments when using `typedoc-plugin-remark`!

### Proposed solution

Any mechanism for disabling these would be fine. A non-breaking change could include adding a new `"disableRemarkPlugins"` field to the config, or maybe just a new `{ disable: true }` option to the options provided in the existing `"remarkPlugins"` field. I don't  have any real preference though!

**Comment by tgreyuk** [comment]
Makes sense. My initial thinking would be an option `defaultRemarkPlugins` which accepts true/false flags. Will be a simple update and will implement a.s.a.p.

```
defaultRemarkPlugins: {
    gfm: true,
    frontmatter: true,
    mdx: true,
 }
```

**Comment by smoores-dev** [comment]
That seems great! Thanks so much for the quick response!

**Comment by tgreyuk** [comment]
@smoores-dev this is available `typedoc-plugin-remark@1.2.0` .

Docs: https://www.typedoc-plugin-markdown.org/plugins/remark/options#--defaultremarkplugins
